api/.github/workflows/lint.yaml

```diff
@@ -1,37 +0,0 @@
-name: Lint
-
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Lint markdown files
-        uses: docker://avtodev/markdown-lint:v1
-        with:
-          args: "./**/*.md -i ./CHANGELOG.md"
-
-      - name: Install dependencies
-        run: npm install
-
-      - name: Build 🔧
-        run: npm run compile
-
-      - name: Lint
-        run: npm run lint
-
-      - name: Check for Circular Dependencies
-        run: npm run cycle-check
```

Comment:
I don't see the cycle check right now in the core repo's lint workflow - is that something we should add as a note to the `TODO.md` file?

Reply:
It was a problem specifically in the API as some of the API internal modules can depend on each other. The main repo splits everything into packages and if there is a cycle installation would fail. I think this isn't a problem but I could be wrong.

Reply:
Alright, thanks for the explanation! :slightly_smiling_face: